netflix/src/components/main/sort/SortByOption.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from "react";
+import { filter } from "../../../redux/films/api";
+import { useSelector, useDispatch, batch } from "react-redux";
+import { setCurrentSort, setFilms } from "../../../redux/films/films";
+import { sortString } from "../../../helpers/sortHelperMethods";
+import { selectStatus } from "../../../redux/selectors/films";
+import { useSearchState } from "../../../hooks/useSearchState";
+
+const SortByOption = ({ option }: any) => {
+  const dispatch = useDispatch();
+  const searchState = useSelector(selectStatus);
```

Comment:
Can you move `const searchState = useSelector(selectStatus);` inside of `useSearchState`?